swarm/network/stream/syncer.go

```diff
@@ -209,46 +199,6 @@ func (s *SwarmSyncerClient) BatchDone(stream Stream, from uint64, hashes []byte,
 	return nil
 }
 
-func (s *SwarmSyncerClient) TakeoverProof(stream Stream, from uint64, hashes []byte, root storage.Address) (*TakeoverProof, error) {
```

Comment:
I am so glad if this goes away....violates our rule of cluttering the code with things which *maybe* could be used in the future...

Reply:
Exactly the point of this PR :) 99% of the time we should not be including code that is not used immediately in the PR that introduces it. There are only very few exceptions when this is helpful.

Reply:
👍